web_dashboard_open_action/README.rst

```diff
@@ -0,0 +1,30 @@
+Open a dashboard's action
+=========================
+
+This module adds a button to open a dashboard's action in the usual view, thus enabling filtering and any other actions you can apply on the resulting records. Basically, this is the inverso to `Add to dashboard`.
+
+Usage
+=====
+
+After installation, a dashboard has a `Maximize`-button that loads the dashboard's action in a normal view.
+
+Credits
```

Comment:
The OCA template has now been extended with a Bug Tracker section containing a template url in which you are supposed to substitute the repo and module names:

https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst